config/arpwatch.xml

```diff
@@ -81,9 +81,12 @@
 	</additional_files_needed>
 	<fields>
 		<field>
-			<fielddescr>Listening Interface</fielddescr>
-			<fieldname>interface</fieldname>
-			<description>Choose the desired listening interface here.</description>
+			<fielddescr>Interfaces</fielddescr>
+			<fieldname>interface_array</fieldname>
```

Comment:
I looked over the code and to change this field back, a new variable would have to be create for $interface i think. I believe they would conflict because the original package would only allow you to select 1 interface. I've used arpwatch for a couple of years and every firmware update that will force a package install, i would have to re-save the interface before the interface would start